src/js/edu-benefits/5490/config/form.js

```diff
@@ -498,11 +502,25 @@ const formConfig = {
                   'ui:options': {
                     hideIf: form => {
                       const status = _.get('highSchool.status', form);
-                      return status !== 'graduated';
+                      return status !== 'graduated' && status !== 'graduationExpected';
                     },
-                    expandUnder: 'status'
-                  }
-                }
+                    expandUnder: 'status',
```

Comment:
Could check to see if `expandUnder` is a string (do the truthy thing like we are now) or a function (pass params to it like `hideIf`...or `updateSchema`, maybe...and then check if the result is truthy).